Rakefile

```diff
@@ -12,7 +12,7 @@ require './card_generator'
 include BoilerMakeBingo
 
 TechWords = %w{JavaScript Ruby Go Swift C Haskell Rust Erlang Elixir TypeScript Elm LISP Node C++ C# Java Python Xamarin Vue Angular React ReactNative Flutter Kotlin Machine\ Learning ARKit AI Python}
-ServiceWords = %w{Twitter Twilio HealthKit IFTTT Fitbit Github Sunlight\ Labs OpenGov ARKit Alexa Google Facebook Minecraft\ Forge}
+ServiceWords = %w{Twitter Twilio HealthKit IFTTT Fitbit Github Sunlight\ Labs OpenGov ARKit Alexa Google Facebook Minecraft\ Forge NOAA\ Climate\ Data\ Online}
```

Comment:
this is likely too long... but I'll try it and see what happens.

Reply:
@jonfuller if it's too long, you can shorten it to `NOAA\ CDO`